src/Kernel/client/ExecutionPathVisualizer/fontSizes.ts

```diff
@@ -0,0 +1,587 @@
+// Retrieved from https://github.com/adambisek/string-pixel-width/blob/master/src/widthsMap.js
```

Comment:
Would it be possible to use the [string-pixel-width](https://www.npmjs.com/package/string-pixel-width) npm package instead of copying this code here?

Reply:
It turns out to be a little squirrly to import npm packages into the Jupyter Notebook client. I came up with one approach at https://github.com/microsoft/iqsharp/commit/510979a1196a63c6773a3a893a062b77909e81bd, but it's not great that it relies on deprecated TypeScript features...

Reply:
Good catch. This was mainly used for determining text width, which was needed for my prototype project in Node,js. But now that it's purely in-browser, we might be able to just use hidden divs as calculate the width from there.

Reply:
I made a new commit where I use in-browser DOM manipulation instead. This would require an additional dev package when we bring my jest tests into this repo.